convert2rhel/unit_tests/logger_test.py

```diff
@@ -184,3 +186,27 @@ def test_archive_old_logger_files(log_name, path_exists, tmpdir, caplog):
 def test_should_disable_color_output(monkeypatch, no_color_value, should_disable_color):
     monkeypatch.setattr(os, "environ", {"NO_COLOR": no_color_value})
     assert logger_module.should_disable_color_output() == should_disable_color
+
+
+@pytest.mark.noautofixtures
+def test_logfile_buffer_handler(read_std):
+    logbuffer_handler = logger_module.LogfileBufferHandler(2, "custom_name")
+    logger = logging.getLogger("convert2rhel")
+    logger.addHandler(logbuffer_handler)
+
+    logger.warning("message 1")
+    logger.warning("message 2")
+
+    # flushing without other handlers should work, it will just go to NullHandlers
+    logbuffer_handler.flush()
```

Comment:
Do you know if there is a way to assert that the NullHandler actually got the messages after the flush? 

Not sure if that would be possible, probably not, right?

Reply:
We can mock the NullHandler but I don't think it's worth the effort given it's supposed to throw it away right away. WDYT?

Reply:
Makes sense. I don't think mocking it will be of any help.